docs/configuration.md

```diff
@@ -86,7 +86,7 @@ be put in the Flink configuration file.
 | `remote-shuffle.transfer.client.num-threads` | Integer | `-1` | 1.0.0 | false | The number of Netty threads to be used at the client (flink job) side. The default `-1` means that `2 * (the number of slots)` will be used. |
 | `remote-shuffle.transfer.client.connect-timeout` | Duration | `2min` | 1.0.0 | false | The TCP connection setup timeout of the Netty client. |
 | `remote-shuffle.transfer.client.connect-retries` | Integer | `3` | 1.0.0 | false | Number of retries when failed to connect to the remote `ShuffleWorker`. |
-| `remote-shuffle.transfer.client.connect-retry-wait` | Duration | `3s` | 1.0.0 | flase | Time to wait between two consecutive connection retries. |
+| `remote-shuffle.transfer.client.connect-retry-wait` | Duration | `3s` | 1.0.0 | false | Time to wait between two consecutive connection retries. |
 | `remote-shuffle.transfer.transport-type` | String | `auto` | 1.0.0 | false | The Netty transport type, either `nio` or `epoll`. The `auto` means "selecting the proper mode automatically based on the platform. Note that the `epoll` mode can get better performance, less GC and have more advanced features which are only available on modern Linux. |
 | `remote-shuffle.transfer.send-receive-buffer-size` | MemorySize | `0b` | 1.0.0 | false | The Netty send and receive buffer size. The default `0` means the system buffer size (cat /proc/sys/net/ipv4/tcp_[rw]mem) and is 4 MiB in modern Linux. |
 | `remote-shuffle.transfer.heartbeat.interval` | Duration | `1min` | 1.0.0 | false | The time interval to send heartbeat between the Netty server and Netty client. |
```

Comment:
Typing errors for `remote-shuffle.transfer.heartbeat.timeout`. The default value should be `false` instead of `flase`.  The similar error is in line 92 and line 191.